storybook/storybook-react/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import { Badge } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Feedback/Badge',
+  component: Badge,
+  args: {
+    label: 'Tip',
+  },
+  argTypes: {
+    color: {
+      control: {
+        type: 'inline-radio',
```

Comment:
I think a select is more user-friendly here

Reply:
Done! And checked all components using a select, radio, or inline radio.